server/app.js

```diff
@@ -40,8 +42,7 @@ app.use(function(err, req, res, next) {
   res.json({ error: err });
 });
 
-//connectDB();
-//app.listen(3000, ()=>console.log('server started on port 3000'));
-//module.exports = app;
+connectDB();
+app.listen(3000, ()=>console.log('server started on port 3000'));
```

Comment:
The listen event already happens somewhere else. See this comment I made on Haruka's PR: https://github.com/hatchways/team-fettuccini/pull/32#discussion_r404508082